decidim-process-extended/lib/decidim/process/extended/process_serializer.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Process
+    module Extended
+      # This class serializes a ParticipatoryProcess so can be exported
+      # to CSV, JSON or other formats (check Decidim::Exporters)
+      class ProcessSerializer < Decidim::Exporters::Serializer
+        # Public: Initializes the serializer with a ParticipatoryProcess.
+        def initialize(process)
+          @process = process
+        end
+
+        # Public: Returns a hash with the serialized data.
+        def serialize
+          {
+            # Process Information
+            id: process.id,
+            title_ca: process.title['ca'],
+            slug: process.slug,
+            short_description_ca: process.short_description['ca'],
+            process_type: process_type,
+            scope_id: process.scope&.id,
+            scope_name_ca: process.scope&.name.try(:[], 'ca'),
+            department_id: process.area&.id,
+            department_name_ca: process.area&.name.try(:[], 'ca'),
+            participatory_space: process.participatory_process_group&.name.try(:[], 'ca')&.downcase,
+            normative_type_id: process.scope&.scope_type&.id,
+            normative_type_name_ca: process.scope&.name.try(:[], 'ca'),
+            duration_days: duration_days,
+            start_date: process.start_date,
+            end_date: process.end_date,
+            cost: process.cost,
+            has_record: process.has_summary_record,
```

Comment:
`process.has_summary_record?` for readability please

Reply:
Thanks